app/controllers/moderation/questions_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Moderation::QuestionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
```

Comment:
would be better to call this method `show` as it needs an id (`index` would be a listing of all the things instead of a specific one)